src/server/pfs/server/api_server.go

```diff
@@ -268,6 +268,28 @@ func (a *apiServer) CopyFile(ctx context.Context, request *pfs.CopyFileRequest)
 	return &types.Empty{}, nil
 }
 
+func (a *apiServer) GetFiles(request *pfs.GetFileRequest, apiGetFilesServer pfs.API_GetFilesServer) (retErr error) {
+	func() { a.Log(request, nil, nil, 0) }()
+	defer func(start time.Time) { a.Log(request, nil, retErr, time.Since(start)) }(time.Now())
+
+	//file, err := a.driver.getFiles(a.getPachClient(apiGetFilesServer.Context()), request.File, request.OffsetBytes, request.SizeBytes)
+	file, err := a.driver.getFile(a.getPachClient(apiGetFilesServer.Context()), request.File, request.OffsetBytes, request.SizeBytes)
+	if err != nil {
+		return err
+	}
+
+	gfrReader, err := a.driver.filesFromByteStream(a.getPachClient(apiGetFilesServer.Context()), file, request.File, request.OffsetBytes, request.SizeBytes)
```

Comment:
TODO: this is taking in `request.SizeBytes` to use to determine the file sizes. This is incorrect and should instead take a sizes based on what happens in `driver.getFile()`. This will slightly change the `driver.getFile()` signature in order to have it return info on what the size of each obj/block that will then be passed to `driver.filesFromByteStream()`.